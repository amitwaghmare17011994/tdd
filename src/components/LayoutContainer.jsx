import React from 'react'
import { Theme, createStyles, makeStyles } from '@material-ui/core/styles';

const useStyles = makeStyles((theme: Theme) =>
    createStyles({
        container: {
            display: 'flex',
            flexDirection: 'column',
            flex: 1,
            justifyContent: 'center',
            height: '100%',
            textAlign: 'center',
            backgroundColor: '#f4f4f4'
        },

    }),
);

const LayoutContainer = (props) => {
    const classes = useStyles()
    return <div className={classes.container}>
        <div style={{ padding: 20,justifyContent:'center',display:'flex' }}>
            {props.children}
        </div>
    </div>

}
export default LayoutContainer