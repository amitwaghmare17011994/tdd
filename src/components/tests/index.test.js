import { create, act } from 'react-test-renderer';
import LayoutContainer from '../LayoutContainer';


describe('Layout Container', () => {

    
    it('should render Layout Container', () => {
        const root = create(<LayoutContainer />)
        expect(root.toJSON()).toMatchSnapshot();

    })

    


})
