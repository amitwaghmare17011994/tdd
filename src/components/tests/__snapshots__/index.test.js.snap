// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout Container should render Layout Container 1`] = `
<div
  className="makeStyles-container-1"
>
  <div
    style={
      Object {
        "display": "flex",
        "justifyContent": "center",
        "padding": 20,
      }
    }
  />
</div>
`;
